src/common/components/elements/FollowerBubble.tsx

```diff
@@ -3,60 +3,32 @@ import Link from 'next/link';
 import Image from 'next/image';
 import styled, { css } from 'styled-components';
 import cx from 'classnames';
-import { Follower } from './FollowerCount';
-import { useState, useEffect, FC } from 'react';
-import { useWalletProfileLazyQuery } from 'src/graphql/indexerTypes';
+import { FC } from 'react';
+import { TopFollower } from '@/common/context/ProfileData';
 
 type FollowerBubbleProps = {
   isFirst?: boolean;
-  follower: Follower;
+  follower: TopFollower;
 };
 
 export const FollowerBubble: FC<FollowerBubbleProps> = ({ isFirst, follower }) => {
-  const { fromHandle } = follower.twitter;
-
-  const [status, setStatus] = useState<'idle' | 'loading' | 'done'>('idle');
-  const [queryWalletProfile, result] = useWalletProfileLazyQuery();
-  useEffect(() => {
-    (async () => {
-      setStatus('loading');
-      try {
-        if (fromHandle) {
-          queryWalletProfile({
-            variables: {
-              handle: fromHandle,
-            },
-          });
-        }
-      } finally {
-        setStatus('done');
-      }
-    })();
-  }, [fromHandle, queryWalletProfile]);
-
-  const isLoading = status === 'loading' || status === 'idle' || result.loading;
-
   return (
-    <Link href={`/profiles/${follower.account.from.toBase58()}`} passHref>
+    <Link href={`/profiles/${follower.pubKey}`} passHref>
       <a
         className={cx({
           'ml-[-8px] block': !isFirst,
```

Comment:
You can use "-space-x-2" on the parent div instead of the pr element styling. 

Here is a full example
![image](https://user-images.githubusercontent.com/10433154/162268475-076000e5-f099-4858-ad2c-f51823f66e7b.png)

 <div ```
className="flex -space-x-2 overflow-hidden">
        <img
          className="inline-block h-10 w-10 rounded-full ring-2 ring-white"
          src="https://images.unsplash.com/photo-1491528323818-fdd1faba62cc?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=facearea&facepad=2&w=256&h=256&q=80"
          alt=""
        />
        <img
          className="inline-block h-10 w-10 rounded-full ring-2 ring-white"
          src="https://images.unsplash.com/photo-1550525811-e5869dd03032?ixlib=rb-1.2.1&auto=format&fit=facearea&facepad=2&w=256&h=256&q=80"
          alt=""
        />
        <img
          className="inline-block h-10 w-10 rounded-full ring-2 ring-white"
          src="https://images.unsplash.com/photo-1500648767791-00dcc994a43e?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=facearea&facepad=2.25&w=256&h=256&q=80"
          alt=""
        />
        <img
          className="inline-block h-10 w-10 rounded-full ring-2 ring-white"
          src="https://images.unsplash.com/photo-1472099645785-5658abf4ff4e?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=facearea&facepad=2&w=256&h=256&q=80"
          alt=""
        />
      </div>
```